### PR TITLE
docs(pantilla-para-aplicacion-web.md): update content

### PR DESCRIPTION
**Pull request by AngeloCG97**
### Improve WebApp Boilerplate Page

#147

### Steps to test
1. Run project locally

#### CheckList
- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish

### CODE REVIEW

File: docs/plantilla-para-aplicacion-web.md [hunk 1]
```diff
@@ -7,6 +7,43 @@ sidebar_label: Plantilla para Aplicación Web
 ## Plantilla para Aplicación Web de React
 
 Plantilla para aplicaciones eb de React incorporando UAL para autenticadores EOSIO (wallets)
```
Comment (leisterfrancisco) [inline docs/plantilla-para-aplicacion-web.md:9]: Error en la palabra **eb**, en la oración "Plantilla para aplicaciones eb de React incorporando UAL para autenticadores EOSIO (wallets)"

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]
Buen trabajo

File: docs/plantilla-para-aplicacion-web.md (hunk 1 above)
Comment (AngeloCG97) [reply docs/plantilla-para-aplicacion-web.md:9]: Thanks Leister, this error fix in other PR please or contributes in this PR and fix error

**Comment by xavier506** [comment]
Closing due to incorrect conent

**Pull request closed without merging**